napari/_vispy/quaternion.py

```diff
@@ -4,6 +4,14 @@
 def quaternion2euler(quaternion, degrees=False):
     """Converts VisPy quaternion into euler angle representation.
 
+    Euler angles have degeneracies, so the output might different
+    from the Euler angles that might have been used to generate
+    the input quaternion.
+
+    Euler angles representation also has a singularity
+    near pitch = Pi/2 ; to avoid this, we set to Pi/2 pitch angles
+     that are closer than the chosen epsilon from it.
```

Comment:
```suggestion
    that are closer than the chosen epsilon from it.
```

Just removing a leading space. I think that's fine with it, but it may be RST will parse that as a nested block.